lib/tests/CMakeLists.txt

```diff
@@ -10,7 +11,7 @@ set(TEST_CASES
     tokenizer_test.cc
     test_utils.cc
     config_document_tests.cc
-    conf_parser_test.cc)
+    conf_parser_test.cc config_substitution_test.cc)
```

Comment:
Bleh. Check if you have the latest CLion, I thought mine was handling this better.